Runtime/Editor/VisualElements/LootLocker MainWindow/LootLockerMainWindow.cs

```diff
@@ -0,0 +1,503 @@
+                keyApply.style.borderTopColor = new Color(0.10980392156862745f, 0.9098039215686274f, 0.42745098039215684f);
+        }
+        else
+        {
+            keyApply.text = "Apply";
+            keyApply.style.borderRightColor =
+            keyApply.style.borderLeftColor =
+            keyApply.style.borderBottomColor =
+            keyApply.style.borderTopColor = new Color(0.0196078431372549f, 0.15294117647058825f, 0.0784313725490196f);
+        }
+        keyApply.clickable.clickedWithEventInfo += ApplyKeyClicked;
+        keyTemplate.Add(keyApply);
+        keyTemplate.AddToClassList("apikey");
+        keyList.Add(keyTemplate);
+    }
+
+    public void PopulateList(ContentType contentType)
+    {
+        optionList.Clear();
+
+        switch (contentType)
+        {
+            case ContentType.Organisations:
+                currentPage = Page.OrganisationPage;
+                listHeader.text = "Organisations:";
+                foreach (var org in activeUser.organisations)
+                {
+                    var btn = GenerateButton(org);
+
+                    btn.clickable.clickedWithEventInfo += OrgButtonClicked;
+
+                    optionList.Add(btn);
+                }
+                break;
+            case ContentType.Games:
+
+                currentPage = Page.GamePage;
+                listHeader.text = "Games:";
+                foreach (var game in activeOrganisation.games)
+                {
+                    var btn = GenerateButton(game, true);
+                    btn.clickable.clickedWithEventInfo += GameButtonClicked;
+                    optionList.Add(btn);
+                }
+
+                returnBtn.text = "Change Organisation";
+                break;
+            case ContentType.GameOptions:
+
+                currentPage = Page.GameOptionPage;
+
+                Button apikeybtn = new Button();
+                apikeybtn.text = "API Keys";
+                apikeybtn.clickable.clicked += OpenAPIKeyTab;
+                apikeybtn.style.flexGrow = 1;
+                apikeybtn.style.flexDirection = FlexDirection.Row;
+                apikeybtn.style.width = 200;
+                apikeybtn.style.height = 50;
+                apikeybtn.AddToClassList("optionbtn");
+                optionList.Add(apikeybtn);
+                break;
+
+
+        }
+    }
+
+    public Button GenerateButton(Game game, bool caret)
+    {
+        Button newButton = new Button();
+        newButton.style.flexGrow = 1;
+        newButton.style.flexDirection = FlexDirection.Row;
+        newButton.name = game.id.ToString();
+        newButton.style.width = 200;
+        newButton.style.height = 50;
+
+
+        Label buttonLabel = new Label();
+        buttonLabel.name = "name";
+        buttonLabel.text = game.name;
+        buttonLabel.style.fontSize = 15;
+
+        newButton.Add(buttonLabel);
+        buttonLabel.AddToClassList("optiontext");
+
+        if (caret)
```

Comment:
Why is the caret only there for games and not organisations?

Reply:
The caret shows that there is options when pressed, whereas organisations don't have options. I can add it if we want of course

Reply:
Aight